docker/supabase/volumes/db/init/04-initiate-tables.sql

```diff
@@ -0,0 +1,14 @@
+CREATE extension IF NOT EXISTS citext  WITH SCHEMA extensions;
+
+CREATE TABLE IF NOT EXISTS telescope_profiles (
+  id text PRIMARY KEY,
+  registered_at timestamp with time zone DEFAULT timezone('utc'::text, now()) NOT NULL,
+  updated_at timestamp with time zone DEFAULT timezone('utc'::text, now()) NOT NULL,
+  feeds text[] NOT NULL,
+  display_name citext UNIQUE NOT NULL,
+  first_name text NOT NULL,
+  last_name text NOT NULL,
+  email citext UNIQUE NOT NULL,
```

Comment:
https://github.com/Seneca-CDOT/telescope/blob/master/docker/supabase/volumes/db/init/01-auth-schema.sql#L11

Reply:
We could do this but later we'll have to call `toLower` to make sure the email is unique because emails are case-insensitive.

Reply:
I don't think we care, do we?  The email is informational only.  The hashed email (`id`) is all we really care about.  But either way is fine.